### ISSUE TITLE
Add VHS as Source/Quality

### ISSUE THREAD

**Issue by AnnieTheEagle** [Radarr/Radarr#2551]
**Description:**
Certain very old or obscure movies (such as Ricky 1) are only available on VHS. Would it be possible to add a quality called "VHS" and perhaps rank it just above "Unknown"?

**Radarr Version:**
 0.2.0.910

**Mono Version:**
Mono JIT compiler version 4.6.2 (Debian 4.6.2.7+dfsg-1)

[AB#865](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/865)

**Comment by dutch2005** [comment]
Cant ya just add a profile rule to have this sorted?

https://en.wikipedia.org/wiki/VHS

NTSC VHS is roughly equivalent to 333×480 pixels luma and 40×480 chroma resolutions (333×480 pixels=159,840 pixels or 0.16MP (1/6 of a MegaPixel)).,[36] while PAL VHS offers the equivalent of about 335×576 pixels luma and 40×240 chroma (the vertical chroma resolution of PAL is limited by the PAL color delay line mechanism).

TL;DR VHS is converted 480p or 576p

**Comment by deadman36g** [comment]
I agree. VHS quality (perhaps even Laser Disc) should be added as a quality, perhaps directly under DVD.

**Comment by bakerboy448** [comment]
Examples

Das.Schweigen.der.Hammel.1994.GERMAN.VHSRiP.x264-TAPECRACKERs
A.Morgadinha.dos.Canaviais.1949.VHSRip.Xvid-UpPTMSNM
Shes.In.The.Army.Now.1981.VHSRip.x264-bigbumbee
Clarence.1990.VHSrip.XviD-CG

**Comment by austinwbest** [comment]
Thetr won't be more qualities added but should be mapped to SD instead of Unknown

**Comment by dutch2005** [comment]
If this is to be implented, one would need to add VHS-PAL and VHS-NTSC

As there is a slight difference between the 2

VHS NTSC = 30 frames /sec
VHS PAL = 25 frames /sec

VHS NTSC = 525p
VHS PAL = 625p

Difference in resolution

Since PAL does not produce as much frames as NTSC, there is a significant difference between the resolution of PAL and the resolution of NTSC. Therefore, the difference in resolution is the most profound difference.

The number of resolution lines that NTSC broadcasts is 525 lines while PAL broadcasts 625 lines of resolution. What difference does this make? There is often a positive correlation between resolution and visual information or between resolution and video quality. The more lines of resolution there is the higher the amount of visual information and vice versa. Hence, PAL produce better image quality and gives our vision a higher amount of visual information.

**Comment by tommitytom** [comment]
Is there a way of adding this manual via "Custom Formats" or something?  I can't quite work it out...

**Comment by bakerboy448** [comment]
> Is there a way of adding this manual via "Custom Formats" or something? I can't quite work it out...

No.

Please use one of the support channels: [Reddit](https://www.reddit.com/r/radarr/) or  [Discord](https://radarr.video/discord) for support/questions.

**Comment by bakerboy448** [comment]
closing in favor of #7717
